compiler/bcq-tools/README.md

```diff
@@ -76,3 +76,38 @@ const(value=[1, 2, 3, -3], name='const3')
 ### Caution
 
 - Generated output_arrays will be start with comma.
+
+## generate_bcq_metadata
+
+### Purpose
+
+To apply BCQ, additional information which is not included in original model is needed.
+For example, without metadata, we cannot know from where to where BCQ related nodes are included. Metadata for BCQ is needed due to those reason and `generate_bcq_metadata` will generate metadata in the model according to each BCQ version.
```

Comment:
if `metadata` is addition information, it would be better to add this explanation.

Reply:
`from where to where`; is this like word to word translation from Korean?
I'm not sure the meaning is unstandable

Reply:
`For example` : if this is one of several examples, using `for example` seems suitable. If not, I think `For example` can be omitted.

Reply:
Can this be what your intension is?
> `generate_bcq_metadata` is to append metadata in BCQ model that is used to connect BCQ related operators and constant nodes.

Reply:
> if metadata is addition information, it would be better to add this explanation.

OK, I will add it.

> from where to where; is this like word to word translation from Korean? I'm not sure the meaning is unstandable

I searched about this expression and it seems be used in English but... I am not sure :(

> For example : if this is one of several examples, using for example seems suitable. If not, I think For example can be omitted.

Metadata has more information than the example :)

> Can this be what your intension is?
> generate_bcq_metadata is to append metadata in BCQ model that is used to connect BCQ related operators and constant nodes.

Quite simillar :)


I will fix the sentence with applying your opinion!